zubhub_frontend/zubhub/src/components/form/tagsInput/TagsInput.jsx

```diff
@@ -115,14 +156,15 @@ export default function TagsInput({
         <Box style={{ position: 'relative' }}>
           <Box className={classes.suggestionBox}>
             <ClickAwayListener onClickAway={handleClickAway}>
-              <Box>{remoteData.length > 0 ? remoteData : <p>Item not found: Hit Enter to Save your Input</p>}</Box>
+              <Box>{remoteData.length > 0 ? remoteDataRender : <p>Item not found: Hit Enter to Save your Input</p>}</Box>
             </ClickAwayListener>
           </Box>
         </Box>
       ) : null}
 
+      {errorMessage && <p style={{ color: 'red' }}>{errorMessage}</p>}
```

Comment:
use a common class for errors

Reply:
See existing ways in which errors are displayed